x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/http/retry/ErrorResponse.java

```diff
@@ -22,7 +22,7 @@ public ErrorResponse(String errorMessage) {
         this.errorStructureFound = true;
     }
 
-    private ErrorResponse(boolean errorStructureFound) {
+    protected ErrorResponse(boolean errorStructureFound) {
```

Comment:
Changing to `protected` because `UnifiedChatCompletionErrorResponse` needs to call it.